### PR TITLE
Miscellaneous Housekeeping Tasks

### PR DESCRIPTION
**Pull request by natalialuzuriaga**
## Problem

After reviewing the repository, made edits to various areas of the ospo guide!

## Solution
- File names are consistent across all files -- lowercase with hypens
- Logos of early career talent programs have been added.
- Updated VDP text in README
- Removed cookiecutter variables with project info

### CODE REVIEW

File: resources/xz-supply-chain-attack.md
Comment (decause-gov) [inline resources/xz-supply-chain-attack.md:1]: hate to do it, but we already shipped this URL a bunch of places, so let's keep this one the same :( If there was a way to use a redirect, I would recommend that. This is why we are excited for go/links ;)

**Review by decause-gov — CHANGES_REQUESTED** [review]
+1 on all changes except for renaming the XZ report. We already shipped that URL to a bunch of places, so sadly, it should remain for now.

File: resources/xz-supply-chain-attack.md
Comment (natalialuzuriaga) [reply resources/xz-supply-chain-attack.md:1]: Oooo true, forgot about that. Reverted

**Review by decause-gov — APPROVED** [review]
LGTM +1 🚢 🧹

**Pull request merged**